### ISSUE TITLE
Core principles of Flutter

### ISSUE THREAD

**Issue by miguelmartins17** [dwyl/flutter-todo-list-tutorial#8]
Where should we include the core principles of Flutter to explain more about how Flutter works and to explore the advantages of using it?

**Comment by nelsonic** [comment]
@miguelmartins17 this goes in `learn-flutter`. 
_Everything_ that is not _specific_ to the Flutter Todo List should go in `learn-flutter`. 
Then we _link_ to `learn-flutter` from the Flutter Todo List.

**Comment by miguelmartins17** [comment]
Of course. Thank you. I'll keep trying to summarize Flutter's best practices.
